### ISSUE TITLE
Limit number of results in query?

### ISSUE THREAD

**Issue by ETGgames** [imperiumlabs/GeoFirestore-iOS#1]
Since now it's build in firestore, and you can do multiple query parameters, is there a possibility that you can add a feature that allows us to limit the number of matches within the area?

**Comment by rtemple6** [comment]
Inside Geofirestore.swift there is a method fireStoreQueryForGeoHashQuery(query:) where you could potentially add (limit:).  However since you are locally modifying the library this probably isn't best approach.

```
internal func fireStoreQueryForGeoHashQuery(query: GFGeoHashQuery) -> Query {
        return self.geoFirestore.collectionRef.order(by: "g").whereField("g", isGreaterThanOrEqualTo: query.startValue).whereField("g", isLessThanOrEqualTo: query.endValue)
    }
```

It would be nice to have the ability to add additional whereField() and limit() to this query!

**Comment by rtemple6** [comment]
#4 @ETGgames  Created a PR for this feature.

**Comment by ETGgames** [comment]
@rtemple6 why use is greater than and less than instead of limit? Also do you think it's possible to query by multiple fields if i add them (sneakily) to the same docs where the location is stored?

**Comment by rtemple6** [comment]
@ETGgames Not entirely sure I understand your first question. The code snippet below is currently how GeoFirestore is fetching from Firestore. 
`internal func fireStoreQueryForGeoHashQuery(query: GFGeoHashQuery) -> Query {
        return self.geoFirestore.collectionRef.order(by: "g").whereField("g", isGreaterThanOrEqualTo: query.startValue).whereField("g", isLessThanOrEqualTo: query.endValue)
    }`

Since this is a regular Firestore Query you can add other fields/limits to it as you normally would any other Firestore query.

**Comment by dhruvshah1214** [comment]
Closing as solved by #4.

**Comment by dbake** [comment]
This seems to be reverted because I don't have the method to limit or access to the limit variable. I also saw that the second commit in #4  seemed to remove the limit functionality
